.travis.yml

```diff
@@ -1,13 +1,10 @@
 language: ruby
 rvm:
-  - 2.2
-  - 2.3
-  - 2.4
```

Comment:
Do you need to delete these versions? We still can support them, I guess. Ruby 2.2 in unsupported anymore, but 2.3 and 2.4 are supported by MRI.

Reply:
I thought activesupport ~> 6.0 requires ruby 2.5 or above.

Reply:
Okay, I didn't know that.